cli/module_generate/_templates/go/models/tmpl-module.go

```diff
@@ -0,0 +1,85 @@
+package models
+
+import (
+    "go.viam.com/rdk/{{ .ResourceType }}s/{{ .ResourceSubtype}}"
+)
+
+var (
+	{{.ModelPascal}} = resource.NewModel("{{ .Namespace}}", "{{ .ModuleName}}", "{{.ModelName}}")
+	errUnimplemented = errors.New("unimplemented")
+)
+
+func init() {
+	resource.Register{{ .ResourceType}}({{.ResourceSubtype}}.API, {{.ModelPascal}},
+		resource.Registration[{{if eq .ResourceSubtype "generic"}}resource.Resource{{else}}{{if eq .ResourceType "component"}}{{.ResourceSubtype}}.{{.ResourceSubtypePascal}}{{else}}{{.ResourceSubtype}}.Service{{end}}{{end}}, *Config]{
+			Constructor: new{{.ModulePascal}}{{.ModelPascal}},
+		},
+	)
+}
+
+type Config struct {
+	// Put config attributes here
```

Comment:
There seem to be a number of places like this where we have instructions for users. Maybe we could add some sort of default text ("user instruction: {...}" or something like that? I dunno) that people could quickly search for to make it easier to know where they need to make updates.

Reply:
I am not opposed to this but in the python stubs we don't have anything that says "user instructions" - so my only concern is that it would make the templates slightly less consistent between languages. what do you think?

Reply:
resolved in person